### PR TITLE
chore: blog post review

### PR DESCRIPTION
**Pull request by theseion**

### CODE REVIEW

**Comment by theseion** [comment]
@airween

**Comment by airween** [comment]
Thanks @theseion!

File: content/blog/2024-08-28-owasp-modsecurity-new-versions.md [hunk 1]
```diff
@@ -17,21 +18,21 @@ It's been a long time since the last releases, especially in case of v2.
...
+* added a new MULTIPART HEADER check - [PR](https://github.com/owasp-modsecurity/ModSecurity/pull/3226)
+* fixed many potential memory leaks and other potential memory handling problems
 
 
 #### Major changes in v3:
 
-* added Windows port - thanks to [@eduar-hte](https://github.com/owasp-modsecurity/ModSecurity/pull/3132)
-* improved CI workflow - also @eduar-hte
-* removed unnecessary string copies, improved engine speed - @eduar-hte
-* fixed a bug in `@pm` operator - [@eduar-hte](https://github.com/owasp-modsecurity/ModSecurity/pull/3233) - please take a look, the operator's behavior has fixed!
```
Comment (airween) [inline content/blog/2024-08-28-owasp-modsecurity-new-versions.md:30]: I agree we don't need to notice users here, but I'm sure we have to notice them some way about the new behavior. Or shouldn't we?

Comment (theseion) [reply content/blog/2024-08-28-owasp-modsecurity-new-versions.md:30]: That's why I had added the line at the top.

**Pull request merged**